src/basic/terminal-util.c

```diff
@@ -1476,3 +1476,15 @@ void get_log_colors(int priority, const char **on, const char **off, const char
                         *highlight = ansi_highlight_red();
         }
 }
+
+int set_terminal_cursor_position(int fd, int row, int column) {
```

Comment:
hmm, i'd really make the row/column parameters `unsigned`. They cannot possibly be negative after all...